.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,11 +18,30 @@ If this pull request addresses a Help Wanted ticket, please link the relevant Gi
 
 Otherwise, link the JIRA ticket.
 -->
+
 #### Release Note
 <!--
-If no release notes are required, just write "NONE" in the release-note block below.
-If yes, a release note is required:
-Enter your past-tense release note in the block below.
+Add a release note for each of the following conditions:
+
```

Comment:
Add a bullet such as: New features and improvements, including behavioural changes, UI changes and CLI changes
Add a bullet such as: Bug fixes and fixes of previous known issues
Add a bullet such as: Deprecation warnings, breaking changes, or compatibility notes

Reply:
@amyblais This would expand upon the scope of the notes I currently provide to you monthly; is that alright with you?

Reply:
The details that I have been asking from you to compile each month -- **config settings, API changes, Websocket changes and Database changes** -- are things that I need from a dev because they're not necessarily obvious from ticket/PR descriptions and I'm not sure where to go in the server repo to compile a list of them myself, so it's also not easy for me to confirm if any of this information is missing from the changelog. These details have always been provided by dev or devops for the changelogs.

The additional bullet points that I'd like to add to the PR template -- **features, bug fixes and breaking changes, etc.** -- are things that devs should also add in PR Release Notes section since we now have a new process where devs sort of write their own release notes that are then added to the changelogs. I haven't asked you to provide these details in the past because it has usually been easy for me to find the details for features and bug fixes myself. I have a process where I regularly go through merged PRs / resolved tickets and then write the changelog based on the information provided in tickets and PR descriptions. I also have a process to ask devs to confirm any breaking changes for each release, e.g. https://community-release.mattermost.com/core/pl/ad7ynbqn5pn1mjsmemsrbg3n1c.

As long as devs make sure that all of the details in the bullet points that will be listed in the PR template are always added in the PR Release Notes section, then likely you won't need to compile a list of **config settings, API changes, Websocket changes and Database changes** in the future since all of that information will be in the PR Release Notes section and easy for me to find for the changelogs.

Hope this helps to clarify things!

Reply:
@amyblais Yes that's very clearly explained, thank you! And thanks for helping me understand that part of the great picture of the release notes :)